src/averageAges.js

```diff
@@ -55,7 +79,23 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const child = people
+    .filter(children => people.some(person => children.mother === person.name));
+
+  const childIsSon = onlyWithSon
```

Comment:
It's not always sons here. Pay attention to variables naming
```suggestion
  const childIsSon = onlyWithSon
```